src/httprpc.cpp

```diff
@@ -101,31 +99,27 @@ static void JSONErrorReply(HTTPRequest* req, UniValue objError, const JSONRPCReq
 
 //This function checks username and password against -rpcauth
 //entries from config file.
-static bool multiUserAuthorized(std::string strUserPass)
+static bool CheckUserAuthorized(std::string_view user_pass)
 {
-    if (strUserPass.find(':') == std::string::npos) {
+    if (user_pass.find(':') == std::string::npos) {
```

Comment:
i intentionally kept this as-is (eg converting to use `Split` slightly changes semantics), but thinking of it, calling `user_pass.find(':')`three times in a row is maybe a bit much.